doc/classes/Tween.xml

```diff
@@ -89,8 +89,8 @@
 		[/codeblocks]
 		Some [Tweener]s use transitions and eases. The first accepts a [enum TransitionType] constant, and refers to the way the timing of the animation is handled (see [url=https://easings.net/]easings.net[/url] for some examples). The second accepts an [enum EaseType] constant, and controls where the [code]trans_type[/code] is applied to the interpolation (in the beginning, the end, or both). If you don't know which transition and easing to pick, you can try different [enum TransitionType] constants with [constant EASE_IN_OUT], and use the one that looks best.
 		[url=https://raw.githubusercontent.com/godotengine/godot-docs/master/img/tween_cheatsheet.png]Tween easing and transition types cheatsheet[/url]
-		[b]Note:[/b] All [Tween]s will automatically start by default. To prevent a [Tween] from autostarting, you can call [method stop] immediately after it is created.
-		[b]Note:[/b] [Tween]s are processing after all of nodes in the current frame, i.e. after [method Node._process] or [method Node._physics_process] (depending on [enum TweenProcessMode]).
+		[b]Note:[/b] Tweens are not designed to be re-used and trying to do so results in an undefined behavior. Create a new Tween for each animation and every time you replay an animation from start. Also Tweens start immediately, so only create a Tween when you want to start animating.
```

Comment:
```suggestion
		[b]Note:[/b] Tweens are not designed to be re-used and trying to do so results in undefined behavior. Create a new Tween for each animation and start the animation immediately. Tweens start animating immediately, so only create a Tween when you want to start animating.
```

Reply:
> Create a new Tween for each animation and start the animation immediately.

This repeats in the next sentence and is misleading, as it's not the user who is starting the animation.

Reply:
Yeah, I don't understand the point of the suggested change.

Reply:
the point is "*in an* undefined behaviour" is strange grammar

Reply:
Okay, but you also removed a huge chunk of the note.

Reply:
I did reword it, I just merged it with the repeated sentence after it.

Reply:
Your suggestion removes important, if a bit verbose, information. I see no point in merging the two sentences and losing the intended message in the process for both of them.